lib/babel.js

```diff
@@ -70,11 +68,43 @@ if (inputSourceMap) {
 }
 
 // include test file
-var transpiled = babel.transformFileSync(testPath, options);
-sourceMapCache[testPath] = transpiled.map;
-requireFromString(transpiled.code, testPath, {
-	appendPaths: module.paths
-});
+var cachePath = hasha(testPath);
+var hashPath = hasha(testPath) + '_hash';
+
+var prevHash = cache.getSync(hashPath, {encoding: 'utf8'});
+var currHash = hasha.fromFileSync(testPath);
+
+if (prevHash === currHash) {
+	var cached = JSON.parse(cache.getSync(cachePath));
```

Comment:
Not sure it is even necessary to make the filename part of the hash. If two files have identical content, it doesn't matter that they are located at different points on the file system - they will encode identically right?